README.md

```diff
@@ -1,11 +1,10 @@
 # Common Hosted Form Service [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE) [![img](https://img.shields.io/badge/Lifecycle-Stable-97ca00)](https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md)
 
+
 ![Tests](https://github.com/bcgov/common-hosted-form-service/workflows/Tests/badge.svg)
 [![Maintainability](https://api.codeclimate.com/v1/badges/950b1d6c61567a1da227/maintainability)](https://codeclimate.com/github/bcgov/common-hosted-form-service/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/950b1d6c61567a1da227/test_coverage)](https://codeclimate.com/github/bcgov/common-hosted-form-service/test_coverage)
 
-Create, edit and publish forms.
-
```

Comment:
The readme _should_ be explaining what CHEFS is/does. Instead of removing this, should we maybe be expanding it to give a better description? Maybe we could find repos with really good readmes and borrow some ideas from them?

This has some interesting info in it too: https://www.appsmith.com/blog/write-a-great-readme